src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -62,14 +63,27 @@ internal class OktaApi : IOktaApi {
 		throw new BmxException( "Error authenticating Okta" );
 	}
 
+	async Task IOktaApi.IssueSmsChallengeMfaOktaAsync( OktaAuthenticateState state, int selectedMfaIndex ) {
+		var mfaFactor = state.OktaMfaFactors[selectedMfaIndex];
+		var authOptions = new AuthenticateChallengeMfaOptions( FactorId: mfaFactor.Id, StateToken: state.OktaStateToken! );
+
+		await _httpClient.PostAsync( $"authn/factors/{authOptions.FactorId}/verify",
+			new StringContent(
+				JsonSerializer.Serialize( authOptions, SourceGenerationContext.Default.AuthenticateChallengeMfaOptions ),
+				Encoding.Default,
+				"application/json" ) );
+	}
+
 	async Task<OktaAuthenticatedState> IOktaApi.AuthenticateChallengeMfaOktaAsync(
 		OktaAuthenticateState state, int selectedMfaIndex,
 		string challengeResponse ) {
 
-
 		var mfaFactor = state.OktaMfaFactors[selectedMfaIndex];
 
-		var authOptions = new AuthenticateChallengeMfaOptions( mfaFactor.Id, challengeResponse, state.OktaStateToken! );
+		var authOptions = new AuthenticateChallengeMfaOptions(
+			FactorId: mfaFactor.Id,
+			StateToken: state.OktaStateToken!,
+			PassCode: challengeResponse );
```

Comment:
The security question MFA uses a field name called ``answer`` instead of ``passcode``, does it also accept the answer as passcode? https://developer.okta.com/docs/reference/api/factors/#request-parameters-23

Reply:
Yup I tested it and it does surprisingly

Reply:
Also currently works on the current BMX as well